Update PowerToys pages

Jay-o-Way opened the pull request:
Most are basic text improvents, some are because of changed UI.
Fixes: microsoft/PowerToys#17268

TO-DO

- [ ] move attributions to (...) index.md header "Contribute" as examples ➡ later
- [x] add Crosshairs image to mouse-utilities.md ➡ later
- [x] re-write pages to adress users with "you"
- [x] check "Get started" --> "Getting started"
- [x] update fmm settings
- [x] update Mouse text on index.md

CC: @crutkas

Jay-o-Way committed:
Update PowerToys pages

Most are basic text improvents, some are because of changed UI.
Clint: I started with mouse pointer crosshairs for https://github.com/microsoft/PowerToys/issues/17268, feel free to add in this section if needed.

CC: @crutkas @dedavis6797 (Deondre, are you still active for PT/docs?(

Jay-o-Way committed:
Merge branch 'MicrosoftDocs:docs' into docs

PRMerger4 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

crutkas commented:
@jay-o-way, i left feedback in your branch.  ping me when you want me to review this

jborsecnik commented:
@Jay-o-Way. Note that this PR can't be merged in draft form (even if it is approved).

Jay-o-Way committed:
adressing feedback

Jay-o-Way committed:
Merge branch 'docs' of https://github.com/Jay-o-Way/windows-uwp into docs

PRMerger16 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Jay-o-Way commented:
@crutkas I'm looking for an image specifically for Crosshairs, but can't seem to find one. Can you add one?

htcfreek commented:
@Jay-o-Way 
The settings table for **find my mouse** is outdated.

Jay-o-Way committed:
More changes

Looked over all the pages. Important changes for Mouse and for Run-Plugin table.

PRMerger12 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

crutkas commented:
- For cross hair, lets make that into a dedicated pr 
- For language, "you" is typically used in context of the end user and this is end user focused, not dev team focused.
- for headings, can you give me more context for where this is.  most of the time we do alphabetical.
- Shortcuts on index.md, i believe we should as this may be the only page someone would visit.  
- Attribution, i agree for being odd in the main page.  I'd actually suggest we have an attribution at the bottom and point to the notice.md file on PowerToys as well as the Community.md file

htcfreek commented on hub/powertoys/install.md:
```diff
@@ -62,24 +57,25 @@ The installer executable accepts the [Microsoft Standard Installer command-line
-| -passive | | progress bar only install |
-| -layout |  | create a local image of the bootstrapper |
-| -log | -l | log to a specific file |
+| Command  | Alternatives | Function     |
```
```suggestion
| Command  | Abbreviation | Function     |
```

Jay-o-Way committed:
Adress people with "you"

Jay-o-Way committed:
Alt -> Abbr

Co-authored-by: Heiko <61519853+htcfreek@users.noreply.github.com>

PRMerger14 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Jay-o-Way commented:
@crutkas

> headings: can you give me more context for where this is.  most of the time we do alphabetical.

There are many different ones. Some regular headers are "Get(ting*) started", "Settings", "Known issues", "Usage"... The modules have specific ones. Most logical would be to look at first time use, or look at the Settings Window. I suppose it's good as is it, just wanted to check consistency across the pages.

\* Wanna use _Get_ or _Getting_?

> Attribution: i agree for being odd in the main page.  I'd actually suggest we have an attribution at the bottom and point to the notice.md file on PowerToys as well as the Community.md file

So, do you want it to be at the bottom of index.md? My idea was to place this in (the bottom of) the page of the module where it actually applies. PT Run is the one and only page that already has this.

crutkas commented:
Constancy is good, "Getting Started" would be what i'd recommend.  

Lets create more targeted issues versus this be a catch all for both the consistence as well as the attribution.

Jay-o-Way committed:
Headers and last updates

Mostly headers.

Jay-o-Way committed:
Merge branch 'docs' of https://github.com/Jay-o-Way/windows-uwp into docs

PRMerger20 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Jay-o-Way commented:
@mattwojo I'm ready :)

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -23,80 +23,74 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
+## PowerToys Settings
 
-| Settings | Description |
+The following general options are available on the PowerToys Settings page.
```
```suggestion
The following general options are available on the PowerToys Run settings page.
```

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -23,80 +23,74 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
-## Settings
 
-The following general options are available for PowerToys Run in the PowerToys settings menu.
+## PowerToys Settings
```
Why should we change this. The new text is incorrect. The settings are for PT Run. And as we are on the Run page I think there is no need to mention it in the header.

Jay-o-Way replied:
I was looking at the different headers on the different pages. Think you've got a point, though...

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -23,80 +23,74 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
-| Activation shortcut | Define the keyboard shortcut to open/hide PowerToys Run |
-| Use centralized keyboard hook | You can try this setting if there are issues with the shortcut 
+| Activation shortcut | Define the keyboard shortcut to show/hide PowerToys Run |
+| Use centralized keyboard hook | Try this setting if there are issues with the shortcut
```
```suggestion
| Use centralized keyboard hook | Try this setting if there are issues with the keyboard shortcut
```

Otherwise it might be translated as file link.

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -23,80 +23,74 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
 
 
-### Direct activation commands
+## Plugins
```
I don't like this change as it was more structured before.

And by the way. The plugins without keyword are not mentioned in the table.

Jay-o-Way replied:
> I don't like this change as it was more structured before.

Sometimes we think the same things, here apparently we don't. I didn't like the current/prevous style. To me, this hierarchy makes sense.

> And by the way. The plugins without keyword are not mentioned in the table.

That's not on me :)

htcfreek replied:
>> And by the way. The plugins without keyword are not mentioned in the table.

> That's not on me :)

I know that. But your headers let the user think the table is a full list of all plugins. With the old layout it was clear that the topic of the table are the action keys and not the list of all plugins.

I think we should add an explaining header or we add all plugins.

htcfreek replied:
I think we should readd the header "direct activation commands" after the text that we have the plugin system. This makes sense as we have this text and tipp and table that are primarily cover the topic direct activation commands.

See my new suggestion.

Jay-o-Way replied:
Hm, Thought I had a good idea. But if you insist on this table being focussed on the DAC's, I will change it back. Note: I will be removing the column "enabled by default" then, because that has nothing to do with the DAC.

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -23,80 +23,74 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
-###  Plugin manager
-
-The PowerToys Run settings menu includes a plugin manager that allows you to enable/disable the various available plugins. By selecting and expanding the sections, you can customize the direct activation commands used by each plugin. In addition, you can select whether a plugin appears in global results, as well as set additional plugin options where available.
+| Plug-in                                                                              | Enabled by default | Direct activation command | Example                                                                                                                                                                                                                                                               ...
```
I don't think the enabled column is as important to be on the second column. please move back

Jay-o-Way replied:
Not very important, but more of a visual thing: "Examples" have even less priority to me; plus, the length of text differs alot and I think that looks better on the right side.

crutkas replied:
I disagree with the reordering here.

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -117,7 +111,7 @@ The Windows System Commands plugin provides a set of system level actions that c
 | `Hibernate` | Hibernates the computer | |
 | `Empty Recycle Bin` | Empties the recycle bin | |
-| `UEFI Firmware Settings` | Reboots the computer into UEFI Firmware Settings | Only available on systems with UEFI firmware.<br />(Requires administrative permissions.) |
+| `UEFI Firmware Settings` | Reboots the computer into UEFI Firmware Settings | Only available on systems with UEFI firmware.<br>(Requires administrative permissions.) |
```
Dont remove / as this is not the correct html code. @crutkas want me to write it with / in an older PR.

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -157,17 +150,17 @@ If the program plugin's option "Include in global result" is not selected, be su
+| Ceiling function | ceil( x.y ) | Rounds a number up to the next larger integer. |
 | Floor function | floor( x.y ) | Rounds a number down to the next smaller integer. |
-| Rounding | round( x.abcd ) | Rounds to the nearest integer.<br />_(Example: `round(8.7867)`)_ |
+| Rounding | round( x.abcd ) | Rounds to the nearest integer. |
```
please don't remove.
this should explain that we mean a decimal number input here

Jay-o-Way replied:
Debatable. I assume one would know what "rounding" means. We also mention the word "integer" already.

htcfreek replied:
With this in mind the example might be to much and it should be clear that `x.abc` means `8.75`.

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -179,7 +172,7 @@ The Calculator plugin supports the following operations:
 | Sign | sign( -x ) | A number that indicates the sign of value:<br />- `-1` if number is less than zero.<br />- `0` if number is zero.<br />- `1` if number is greater than zero. |
 | Random number | rand() | Returns a fractional number between 0 and 1. |
-| Pi | pi | Returns the number of Pi.<br />_(To simply display pi, a sign is required like in this example: `+pi`.)_ |
+| Pi | pi | Returns the number of pi.<br />_(To simply display pi, a sign is required like so: `+pi`.)_ |
```
so => this

htcfreek replied:
Still a grammar issue. 

like **this**:

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -266,11 +256,11 @@ With the Window Walker plugin you can kill the process of a window if it hangs.
 #### File Explorer setting
 
-If the File Explorer settings in Windows are not set to open each window in a seperate process, you will receive the following information when searching for open Explorer windows:
+If the File Explorer settings in Windows are not set to open each window in a seperate process, you will receive the following message when searching for open Explorer windows:
```
why do you change this? information is more clear here that it isn't a warning or error

Jay-o-Way replied:
Because "the message" sounds more like something you actually **can click on**. It's an interactive element.

> After selecting the message, the "Folder options" window will open.

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -106,9 +100,9 @@ The PowerToys Run settings menu includes a plugin manager that allows you to ena
+> If your system language is supported by PowerToys, the system commands will be localized. If you prefer English commands, you can disable the setting **Use localized system commands instead of English ones** in the [plugin manager](#plugin-manager).
 
-| Command / Search result | Action | Note |
+| Command / Search | Action | Note |
```
i think either `command / result` or only `command`

Jay-o-Way replied:
Result are output, not input. I'm going with `command` only then.

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -23,80 +23,74 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
+| Time and date                                                                        | yes                | `(`                       | `( time and date` shows the current time and date in different formats. `( calendar week::04/01/2022` shows the calendar week for the date '04/01/2022'.                                                                                                              ...
+| Time zones                                                                           | yes                | `&`                       | `& Newfoundland` shows the current time in the time zone of Newfoundland.                                                                                                                                                                                             ...
+| Unit converter                                                                       | yes                | `%%`                      | `%% 10 ft in m` to calculate the number of meters in 10 feet.                                                                                                                                                                                                         ...
+| URIs                                                                                 | yes                | `//`                      | `//` to launch your default browser. `// docs.microsoft.com` to have your default browser go to   https://docs.microsoft.com. `mailto:` and `ms-settings:` links are supported.                                                                                       ...
```
please keep the line breakes for better view. that makes it easier to see the relevant information in the text

htcfreek reviewed (changes requested):
You broke many ideas behind the formatting on the run page. I don't like this changes. Please revert. I had some thoughts when doing thing.

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -106,9 +100,9 @@ The PowerToys Run settings menu includes a plugin manager that allows you to ena
 
 > [!TIP]
-> If your system language is supported by PowerToys, the system commands will be localized. If you prefer English commands, you can disable the setting **'Use localized system commands instead of English ones'**  in the [plugin manager](#plugin-manager).
+> If your system language is supported by PowerToys, the system commands will be localized. If you prefer English commands, you can disable the setting **Use localized system commands instead of English ones** in the [plugin manager](#plugin-manager).
```
I added this signs to state that the word have to be seen together as the settings title.

Jay-o-Way replied:
Oh I see. I use the bold text to adress UI elements. Same thing, I guess.

htcfreek replied:
I think we should use both. The sign to encapsulatevthe name parts and the bold format to highlight the settings title

Jay-o-Way replied:
I think both methods try to do the same thing and it's double-up. We don't use the single quotes in any other place.

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -23,80 +23,74 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
 | Ignore shortcuts in Fullscreen mode | When in full-screen (F11), PowerToys Run won't be engaged with the shortcut |
 | Maximum number of results | Maximum number of results shown without scrolling |
 | Clear the previous query on launch | When launched, previous searches will not be highlighted |
 | Preferred monitor position | If multiple monitors are in use, PowerToys Run can be launched on the desired monitor:<br />- Primary display<br />- Display with mouse cursor<br />- Display with focused window |
```
```suggestion
| Preferred monitor position | If multiple displays are in use, PowerToys Run can be launched on the desired display:<br />- Primary display<br />- Display with mouse cursor<br />- Display with focused window |
```

htcfreek reviewed (commented):
to use the same word in all parts of the description.

we use display because it it better to translate

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -23,80 +23,74 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
+## Plugins
 
-These default direct activation commands will force PowerToys Run into only targeted plugins.
+PowerToys Run uses a plugin system to provide different types of results. The default direct activation commands will force PowerToys Run into only targeted plugins.
```
```suggestion
PowerToys Run uses a plugin system to provide different types of results.

### Direct activation commands

The default direct activation commands will force PowerToys Run into only targeted plugins.
```

htcfreek reviewed (commented):
See discussion for line 65

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -23,80 +23,74 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
-
-The PowerToys Run settings menu includes a plugin manager that allows you to enable/disable the various available plugins. By selecting and expanding the sections, you can customize the direct activation commands used by each plugin. In addition, you can select whether a plugin appears in global results, as well as set additional plugin options where available.
+| Plug-in                                                                              | Enabled by default | Direct activation command | Example                                                                                                                                                                                                                                                               ...
+| :----------------------------------------------------------------------------------- | :----------------- | :-------------------------| --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- ...
```
why not `:---`?

Jay-o-Way replied:
No reason at all. Used https://www.tablesgenerator.com/markdown_tables and copy-pasted it.

crutkas commented on hub/powertoys/file-explorer.md:
```diff
@@ -29,25 +29,25 @@ Preview Pane is an existing feature in the Windows File Explorer which shows a l
+![PowerToys Preview Pane demo for Windows 10.](../images/powertoys-fileexplorer.gif)
 
-**Windows 11:**
+**Windows 11**
```
We should move Win11 above Win10 while we are revisiting this

crutkas commented on hub/powertoys/run.md:
```diff
@@ -14,7 +14,7 @@ PowerToys Run is a quick launcher for power users that contains some additional
 
 > [!IMPORTANT]
-> After installing, PowerToys must be enabled and running in the background for this utility to work.
+> PowerToys must be enabled and running in the background for this utility to work.
```
PowerToys needs to be enabled?  that is awkward.  PowerToys must be running as well as PT Run be enabled?

crutkas commented:
I'm afraid this is too big of a PR to keep track of everything and force detailed changes.  can we break this apart into each doc or at least the massive file changes be broken out?  Debating on Run or Fancyzones would derail the other work

Jay-o-Way committed:
PT Run settings

Co-authored-by: Heiko <61519853+htcfreek@users.noreply.github.com>

PRMerger19 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Jay-o-Way committed:
keyboard shortcut

Co-authored-by: Heiko <61519853+htcfreek@users.noreply.github.com>

PRMerger12 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Jay-o-Way committed:
Small / loose changes from feedback

Jay-o-Way committed:
About that table

Jay-o-Way committed:
Remove "PowerToys" from settings headers

Jay-o-Way committed:
Merge branch 'docs' of https://github.com/Jay-o-Way/windows-uwp into docs

PRMerger15 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Jay-o-Way committed:
Keep "Settings" and "Usage" separate

PRMerger5 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Jay-o-Way committed:
missed a few <br>'s

PRMerger3 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Jay-o-Way commented:
PTAL

crutkas commented on hub/powertoys/administrator.md:
```diff
@@ -58,3 +44,21 @@ Admin mode permissions may be required in the following scenarios:
+
+If you see this User Account Control prompt, the application is requesting administrator level elevated permission:
+
+![Windows elevated permission prompt screenshot](../images/pt-admin-prompt.png)
```
i'd pull the images here, they are Win10 ux

Jay-o-Way replied:
Oh well, no problem either, Win.10 isn't dead yet.

crutkas commented on hub/powertoys/administrator.md:
```diff
@@ -58,3 +44,21 @@ Admin mode permissions may be required in the following scenarios:
+
+In the case of an elevated command line, typically the text "Administrator" will be included in the title bar.
+
+![Windows admin command line screenshot](../images/pt-admin-terminal.png)
```
i'd pull the images here, they are Win10 ux

crutkas commented:
Lets merge this and fix the few items we need to in a different PR.

crutkas commented:
@mattwojo waiting for you it looks like

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -23,92 +23,86 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
-| Activation shortcut | Define the keyboard shortcut to open/hide PowerToys Run |
-| Use centralized keyboard hook | You can try this setting if there are issues with the shortcut 
+| Activation shortcut | Define the keyboard shortcut to show/hide PowerToys Run |
+| Use centralized keyboard hook | Try this setting if there are issues with the keyboard shortcut
```
```suggestion
| Use centralized keyboard hook | Try this setting if there are issues with the keyboard shortcut |
```

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -23,92 +23,86 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
+#### Direct activation commands
 
-These default direct activation commands will force PowerToys Run into only targeted plugins.
+A number of the plugins can be activated with a direct activation command so that PowerToys Run will only use the targeted plugin.
```
This is incorrect. You can have an activation command for all plugins. But we don't have one for each by default.

Should be something like: "The plugins... The following table shows the direct activation commands assigned by default."

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -23,92 +23,86 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
-
-![PowerToys Run Plugin Manager](../images/pt-run-plugin-manager.png)
-
+| Plug-in                                                                              | Direct activation command | Example                                                                                                                                                                                                                                                                                    ...
```
Do we mention the plugin or what you can do (`Window Walker` vs `Open windows`)? Should be aligned.

Jay-o-Way replied:
This was written before me, but I can make it fit "plugin" as in Settings window.

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -23,92 +23,86 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
+| Time zones                                                                           | `&`                       | `& Newfoundland` shows the current time in the time zone of Newfoundland.                                                                                                                                                                                                                  ...
+| Unit converter                                                                       | `%%`                      | `%% 10 ft in m` to calculate the number of meters in 10 feet.                                                                                                                                                                                                                              ...
+| URIs                                                                                 | `//`                      | `//` to launch your default browser.<br />`// docs.microsoft.com` to have your default browser go to https://docs.microsoft.com.<br />`mailto:` and `ms-settings:` links are supported.                                                                                                    ...
+| Visual Studio Code<br />_previously opened workspaces, remote machines and containers_ | `{`                     | `{ powertoys` to search for workspaces that contain 'powertoys' in their paths.                                                                                                                                                                                                            ...
```
If we keep description it should be in brakets I think.

Jay-o-Way replied:
Moved these words into the example text.

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -23,92 +23,86 @@ To use PowerToys Run, select <kbd>Alt</kbd>+<kbd>Space</kbd> and start typing! _
+| URIs                                                                                 | `//`                      | `//` to launch your default browser.<br />`// docs.microsoft.com` to have your default browser go to https://docs.microsoft.com.<br />`mailto:` and `ms-settings:` links are supported.                                                                                                    ...
+| Visual Studio Code<br />_previously opened workspaces, remote machines and containers_ | `{`                     | `{ powertoys` to search for workspaces that contain 'powertoys' in their paths.                                                                                                                                                                                                            ...
+| Web search                                                                           | `??`                      | `??` to launch your default browser's search page.<br />`?? What is the answer to life` to search with your default browser's search engine.                                                                                                                                               ...
+| Windows settings                                                                     | `$`                       | `$ Add/Remove Programs` to launch the Windows settings page for managing installed programs.<br >`$ Device:` to list all settings with 'device' in their area/category name.<br />`$ control>system>admin` shows all settings of the path 'Control Panel > System and Security > Administr ...
```
```suggestion
| Windows settings                                                                     | `$`                       | `$ Add/Remove Programs` to launch the Windows settings page for managing installed programs.<br />`$ Device:` to list all settings with 'device' in their area/category name.<br />`$ control>system>admin` shows all settings of the path 'Control Panel > System and Security > Administrative Tools'. |
```

htcfreek reviewed (changes requested):
Small nits and a few decisions needed.

Jay-o-Way committed:
Latest small nits

Feedback from htcfreek;
Alt-text for images: the one for AOT was probably copied from Color Picker;
underscore

Co-Authored-By: Heiko <61519853+htcfreek@users.noreply.github.com>

Jay-o-Way committed:
Latest small nits

Feedback from htcfreek

Jay-o-Way committed:
Alt-text for images

the one for AOT was probably copied from Color Picker

PRMerger13 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -117,7 +111,7 @@ The Windows System Commands plugin provides a set of system level actions that c
 | `Hibernate` | Hibernates the computer | |
 | `Empty Recycle Bin` | Empties the recycle bin | |
-| `UEFI Firmware Settings` | Reboots the computer into UEFI Firmware Settings | Only available on systems with UEFI firmware.<br />(Requires administrative permissions.) |
+| `UEFI Firmware Settings` | Reboots the computer into UEFI Firmware Settings | Only available on systems with UEFI firmware. Requires administrative permissions. |
```
```suggestion
| `UEFI Firmware Settings` | Reboots the computer into UEFI Firmware Settings | Only available on systems with UEFI firmware. (Requires administrative permissions.) |
```

Jay-o-Way replied:
Why are these so important? It's already a "note".

htcfreek replied:
I think this is a side note wich note. The function imo is more important. So the brakets split function description and note.

htcfreek replied:
Ooh. Sory. Didn't saw that it is in the last row. So it ojay as is

htcfreek commented on hub/powertoys/run.md:
```diff
@@ -179,7 +172,7 @@ The Calculator plugin supports the following operations:
 | Sign | sign( -x ) | A number that indicates the sign of value:<br />- `-1` if number is less than zero.<br />- `0` if number is zero.<br />- `1` if number is greater than zero. |
 | Random number | rand() | Returns a fractional number between 0 and 1. |
-| Pi | pi | Returns the number of Pi.<br />_(To simply display pi, a sign is required like in this example: `+pi`.)_ |
+| Pi | pi | Returns the number of pi. (To simply display pi, a sign is required like so: `+pi`.)_ |
```
```suggestion
| Pi | pi | Returns the number of pi. (To simply display pi, a sign is required like so: `+pi`.) |
```

Jay-o-Way committed:
underscore

Co-authored-by: Heiko <61519853+htcfreek@users.noreply.github.com>

PRMerger4 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

htcfreek reviewed (approved):
LGTM

Jay-o-Way committed:
Merge branch 'docs' of https://github.com/Jay-o-Way/windows-uwp into docs

PRMerger18 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Jay-o-Way commented:
@mattwojo Is it expected I update (things like) metadata `ms.date` too?

mattwojo commented:
@Jay-o-Way - Any time that a doc is updated, it's good to update the ms.date to reflect that. If you're using VS Code, you may want to try out the DocsAuthoring extension: https://docs.microsoft.com/en-us/contribute/get-started-setup-tools#install-docs-authoring-pack
It will provide little helpers specific to docs.microsoft publishing, including a pop-up asking if you'd like it to automatically update that ms.date field any time that a change is made to the file. :)
Not required, just a helpful tool that I thought I'd mention.

htcfreek commented:
> @Jay-o-Way - Any time that a doc is updated, it's good to update the ms.date to reflect that. If you're using VS Code, you may want to try out the DocsAuthoring extension: https://docs.microsoft.com/en-us/contribute/get-started-setup-tools#install-docs-authoring-pack
> It will provide little helpers specific to docs.microsoft publishing, including a pop-up asking if you'd like it to automatically update that ms.date field any time that a change is made to the file. :)
> Not required, just a helpful tool that I thought I'd mention.

Neeeed that. Especially the preview feature.

mattwojo commented:
Oh good! My apologies for not sharing that resource with y'all sooner! Happy to help if anyone gets stuck or has questions in regard to using that DocsAuth tool... or in regard to contributing to the docs in general. > mattwoj@microsoft.com

mattwojo commented:
In regard to this PR -- please just @ me when it's ready for a final review and merge.

Jay-o-Way committed:
That Docs Authoring Pack is nice!

Jay-o-Way commented:
@mattwojo Thanks for that plugin! I am indeed (since not so long) using VS Code because Visual Studio is too heavy. I found a few MS plugins already, but this one is a _must_ indeed!
**I'm ready now.**

PRMerger3 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Jay-o-Way committed:
Add PT Crosshairs image

PRMerger16 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Jay-o-Way committed:
...and show image in page

PRMerger17 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Jay-o-Way committed:
Trim border off image

PRMerger3 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

mattwojo commented:
@Jay-o-Way - Is this PR ready to be merged?

Jay-o-Way commented:
> @Jay-o-Way - Is this PR ready to be merged?

Yes. I'm ready 👍🏻
